snowpack/src/build/build-import-proxy.ts

```diff
@@ -235,17 +235,15 @@ export async function wrapImportProxy({
   hmr: boolean;
   config: SnowpackConfig;
 }) {
-  const {baseExt, expandedExt} = getExt(url);
-
   if (typeof code === 'string') {
-    if (baseExt === '.json') {
+    if (url.endsWith('.json')) {
```

Comment:
Overall +1 to removing the `getExt` util—I think it was time for it to go. But one behavior this changes is case—before we transformed everything to lowercase; now if a file is named `FILE.JSON` it won’t be treated as JSON.

Reply:
ah good catch! Maybe instead of removing, we just replace with some simple helper functions `getExt` and `hasExt` to replace `path.extname` and `.endsWith()` repectively

Reply:
I like that. Comment applies everywhere—in this PR we always assume it will be lowercase which will be most of the time but not 100%